contrib/xbacklight_linux.lua

```diff
@@ -0,0 +1,26 @@
+local tonumber = tonumber
+local io = { popen = io.popen }
+local setmetatable = setmetatable
+
+-- xbacklight: provides backlight level
+-- vicious.contrib.xbacklight
+local xbacklight = {}
+
+local function xbacklight_cmd(args)
+    local f = io.popen("xbacklight "..args)
+    if f == nil then
+      return nil
+    else
+      local line = f:read("*all")
+      f:close()
+      return line
+    end
+end
+
+local function worker(format, warg)
+    if not warg then warg = {} end
+    if type(warg) ~= "table" then warg = { warg } end
+    return { math.floor(tonumber(xbacklight_cmd(table.concat(warg, " ")))) }
```

Comment:
`type` and `math.floor` should also be localized like `tonumber` and `io.popen` (the later should be deprecated though)